core/src/i18n/en.yaml

```
@@ -1722,6 +1723,12 @@ subscriptions:
   description: <0>Subscription</0> is used to subscribe to events.
   errors:
     empty-event-types: You must provide at least one event type.
+    event-name-required: Event name is required.
+    event-name-segments: Event name must consist of at least 2 non-empty segments, separated by "." character.
```

Wouldn't that be simplier?
```suggestion
    event-name-segments: Event name must consist of at least 2 non-empty segments, separated by a dot.
```